app/views/converts/new.html.erb

```diff
@@ -0,0 +1,28 @@
+<% title "convert" %>
+<% content_for(:header) do %>
+  <h2>convert splits to other cool formats</h2>
+<% end %>
+<article>
+  <form id="convert" action="/api/v4/convert" method="post" enctype="multipart/form-data">
```

Comment:
cool i am for it, however if a bad file is uploaded the user will see raw json as an error message.

we should either make this a JS submission so that the responses are handled and shown to the user nicely (e.g. like `app/assets/javascripts/upload.coffee` does) or make a separate controller for the user-facing convert feature so that we render html instead of json.